cookbook/recipes/agent.rb

```diff
@@ -0,0 +1,55 @@
+include_recipe 'baragon::common'
+
+["#{node[:baragon][:proxy_conf_dir]}/proxy",
+ "#{node[:baragon][:upstream_conf_dir]}/upstreams"].each do |dir|
+  directory dir do
+    recursive true
+  end
+end
+
+creds = Chef::EncryptedDataBagItem.load('secrets',
+                                        'aws_credentials')['Baragon']
+
+baragon_agent_jar = 'BaragonAgentService-0.1.5-SNAPSHOT.jar'
```

Comment:
we might want to make the baragon version a node setting rather than a hard coded value